### PR TITLE
Added theme colors

### LINKED ISSUE #273: Add file for consistent theme colors.
**Additional Comments**
use colors from #260 
this issue is a "sub task" of #260

### PR DESCRIPTION
**Pull request by ClientCrash**
**Description**
This PR creates a scss file with all theme colors from 
#260 
Close #273

### CODE REVIEW

**Comment by ClientCrash** [comment]
@MaxiHuHe04

**Comment by Akida31** [comment]
probably link #273 instead of #272 in PR description

File: src/app/design/_theme-colors.scss [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/*primary*/
+$primary-default: #2ECC70;
+$primary-hover: #1F8C4C;
+$primary-disabled:#2E473E;
```
Comment (MarcelCoding) [inline src/app/design/_theme-colors.scss:4]:
```suggestion
$primary-disabled: #2E473E;
```

File: src/app/design/_theme-colors.scss [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+/*success*/
+$success-default: #3ABB18;
+$success-hover: #277A10;
+$success-disabled:#2F5629;
```
Comment (MarcelCoding) [inline src/app/design/_theme-colors.scss:8]:
```suggestion
$success-disabled: #2F5629;
```

File: src/app/design/_theme-colors.scss [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+/*warning*/
+$warning-default: #F1C30E;
+$warning-hover: #B3910B;
+$warning-disabled:#675926;
```
Comment (MarcelCoding) [inline src/app/design/_theme-colors.scss:12]:
```suggestion
$warning-disabled: #675926;
```

File: src/app/design/_theme-colors.scss [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+/*danger*/
+$danger-default: #F00040;
+$danger-hover: #B0002F;
+$danger-disabled:#671E35;
```
Comment (MarcelCoding) [inline src/app/design/_theme-colors.scss:16]:
```suggestion
$danger-disabled: #671E35;
```

File: src/app/design/_theme-colors.scss [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+/*info*/
+$info-default: #3281F6;
+$info-hover: #245EB5;
+$info-disabled:#2D456C;
```
Comment (MarcelCoding) [inline src/app/design/_theme-colors.scss:20]:
```suggestion
$info-disabled: #2D456C;
```

**Review by MarcelCoding — CHANGES_REQUESTED** [review]

**Review by MarcelCoding — DISMISSED** [review]

File: src/styles.scss [hunk 6]
```diff
@@ -3,6 +3,7 @@
 @import "variables";
 @import "controls";
 @import "context-menu";
+@import "theme-colors";
```
Comment (MarcelCoding) [inline src/styles.scss:6]: If you don't use it, don't include it.

**Review by MarcelCoding — CHANGES_REQUESTED** [review]

**Comment by MarcelCoding** [comment]
You removed the changes from these comments: https://github.com/cryptic-game/frontend/pull/272/files/32d31bc8a871f63743e608d9e573070497bec3b4

**Comment by ClientCrash** [comment]
> You removed the changes from these comments: https://github.com/cryptic-game/frontend/pull/272/files/32d31bc8a871f63743e608d9e573070497bec3b4

I made changes. @MarcelCoding

**Review by MarcelCoding — APPROVED** [review]

**Comment by ClientCrash** [comment]
@MarcelCoding please request a review from @MaxiHuHe04

**Review by Tert0 — APPROVED** [review]

**Comment by ClientCrash** [comment]
@MaxiHuHe04 is still requested for review. @MarcelCoding can you remove the review request i asked for?

**Comment by ClientCrash** [comment]
@cryptic-game/head-of-frontend @cryptic-game/head-assistant-of-frontend

**Comment by ClientCrash** [comment]
@MaxiHuHe04

**Pull request merged**